src/main/java/seedu/address/model/person/TagEqualsPredicate.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.model.person;
+
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person} has a tag that matches the given tag name.
+ */
+public class TagEqualsPredicate implements Predicate<Person> {
+    private final String tagName;
+
+    public TagEqualsPredicate(String tagName) {
+        this.tagName = tagName;
+    }
+
```

Comment:
Maybe can use .toLowerCase() to set it to lowercase.

Reply:
Thanks for pointing I have added the feature to existing code.